### ISSUE TITLE
group norm support?

### ISSUE THREAD

**Issue by danFromTelAviv** [NVIDIA/apex#602]
Does this also support groupnorm ( instead of batch norm )?

**Comment by alexbrowngh** [comment]
Same here. How can we support GroupNorm?

**Comment by blueardour** [comment]
+1 mark

**Comment by tstandley** [comment]
Wow, is this still open? +1 from me.

**Comment by bonlime** [comment]
Now after Google [released its Bit-S and Bit-M models](https://github.com/google-research/big_transfer) trained with GroupNorm having a memory-efficient and fast version of it would be very nice. Current PyTorch implementation is ~3x times slower than BN and requires much more memory.  
+1 from me

**Comment by directorscut82** [comment]
+1
